src/menus/TrackMenus.cpp

```diff
@@ -74,7 +74,7 @@ void DoMixAndRender(AudacityProject &project, bool toNewTrack)
       }
 
       // Add new tracks
-      const bool stereo = newTracks->NChannels() > 1;
+      const bool stereo = (*newTracks->begin())->NChannels() > 1;
```

Comment:
So now `newTracks.size() == 1` ?

Reply:
Yes

In fact, I now think this commit needs to be in the previous PR for correctness of all stages of the transformation.

So I squashed it into a68333aad87a7dd417aa8ad73e5c8a8c3f8b5bbc